src/gui/input_mac.cpp

```diff
@@ -42,19 +43,45 @@ QString GetModifierPrefix(Qt::KeyboardModifiers mod) noexcept
 	return modprefix;
 }
 
+// Fixup Modifiers, Apples OPTION key
+Qt::KeyboardModifiers de_DE(Qt::KeyboardModifiers mod, QChar c) noexcept
```

Comment:
Let's use a more descriptive name.

`FixInputForGermanLayouts`?

Reply:
i thought of sticking to de_DE and so on might be easier to handle if there is others comming up like pt_PT (portugal) and so on.